.editorconfig

```diff
@@ -202,6 +202,8 @@ resharper_csharp_accessor_owner_body=expression_body
 resharper_redundant_case_label_highlighting=do_not_show
 resharper_redundant_argument_default_value_highlighting=do_not_show
 
+dotnet_diagnostic.IDE0057.severity = none
```

Comment:
```suggestion
dotnet_diagnostic.IDE0057.severity = suggestion
```

Reply:
I went for none since the range operator can't be used in NET462 and NETSTANDARD2.0 (without a polyfill).